ckanext/opendatani/templates/report/view.html

```diff
@@ -0,0 +1,26 @@
+{% extends "page.html" %}
+
+{% block title %}{{ 'Publisher Report' }} - {{ _('Reports') }} - {{ super() }}{% endblock %}
+
+
+{% block breadcrumb_content %}
+  {{ h.build_nav('home', _('Home')) }}
+{% endblock%}
+
+{% for csv_id in extra_vars %}
```

Comment:
Why do we need a loop here? Think you can directly use `csv_id` in html

Reply:
This was another thing that wasn't working initially. I researched a bit, and found that some people needed to loop for variables in dict form. I just checked again with a fresh boot, without the loop, and you're right.